freezeyt/freezing.py

```diff
@@ -171,3 +175,11 @@ def get_links_from_node(node: xml.dom.minidom.Node, base_url) -> list:
     for child in node:
         result.extend(get_links_from_node(child, base_url))
     return result
+
+def check_mimetype(filename, headers):
+    f_type, f_encode = guess_type(str(filename))
+    headers = Headers(headers)
+    cont_type, cont_encode = parse_options_header(headers.get('Content-Type'))
+    if f_type.lower() != cont_type.lower():
+        raise ValueError("Content-type is different to filetype")
```

Comment:
I've tried the second version, than I made rebase about five times but it looks fine, I hope